src/main/resources/static/js/main.js

```diff
@@ -150,47 +149,65 @@
         latitude: position.coords.latitude,
         longitude: position.coords.longitude
       });
+      geometry=true;
       locationButton.toggleClass('loading');
     }, function () {
       locationButton.toggleClass('loading');
       locationButton.toggleClass('disabled');
     })
   });
 
-  const geocodingButton = $('.ui.button#geocoding'); // it is not used
   const card = $('#pac-card');
   const input = document.getElementById('pac-input');
-  //const input = $('.ui.input#pac-input');
   const infowindowContent = document.getElementById('infowindow-content');
-  //const infowindowContent = $('#infowindow-content');
+  const politicalBoundary = 'political'; // Bee.place just contains Administrative/Political location
```

Comment:
There is no `Bee.place` at this moment. What are you referring to?

Reply:
IIRC correctly we agree that Bee Model will contain the following properties:

- name
- mail
- latitude
- longitude
- place
- timezone